src/devicecontroller/DefaultDeviceController.ts

```diff
@@ -374,10 +434,18 @@ export default class DefaultDeviceController implements DeviceControllerBasedMed
     } else {
       tracksToStop = mediaStreamToRelease.getTracks();
     }
+
+    // similar to WebAudio's stream, we need to release the actual input stream to really stop it.
+    if (mediaStreamToRelease === this.chosenVideoTransformDevice?.outputMediaStream) {
+      this.chosenVideoTransformDevice.stop();
```

Comment:
`await`?

Reply:
But hang on — calling `stop` here renders the device unusable in the future. If a developer has a handle on this device and tries to re-select it, what happens?